.eslintrc.json

```diff
@@ -0,0 +1,37 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
+  },
+  "parser": "@babel/eslint-parser",
+  "parserOptions": {
+    "requireConfigFile": "false"
+  },
```

Comment:
- [ ] [REQUIRED] The `.eslintrc.json` configuration file differs slightly from the [config file](https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json) provided for this activity. Please ensure there are no custom changes to the config file.

**Your file**
![tafara-lint](https://user-images.githubusercontent.com/8092327/165364486-0f161a90-0434-4804-9040-2219341add9a.png)

**Original file**
![micro-lint](https://user-images.githubusercontent.com/8092327/165363682-10eeff6f-2d0f-40ac-88e8-9a52b9529d50.png)